pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -1407,6 +1409,7 @@ var _ = Describe("Generating fluentd config", func() {
 						client_cert '/var/run/ocp-collector/secrets/my-infra-secret/tls.crt'
 						ca_file '/var/run/ocp-collector/secrets/my-infra-secret/ca-bundle.crt'
 						type_name _doc
+                        http_backend typhoeus
```

Comment:
formatting

Reply:
@ewolinetz This is clearly a GH rendering issue. :man_shrugging: